wordpress-mobile/WordPress-iOS
# Media Library #3: Deletion

This PR adds the ability to delete media items from the Media Library. There are two ways to do it:

1. View the detail screen for any media item and tap the trashcan in the navigation bar. Tap the Delete button, and the item will be deleted.
2. On the media library grid screen, tap the Edit button in the navigation bar, and then select a number of media items. Tap the trashcan to delete them.

**Note:** This PR currently references a MediaPicker-iOS branch which contains a couple of necessary tweaks. Once that's approved, I'll push out a new pod version and update this branch accordingly.

**To test:**

* Perform the steps above. Check on both dotcom and self-hosted.
* Check the collection view and navigation bar update their state correctly as you enter and exit from Edit mode.
* Check nothing crashes if you delete an item whilst its full size image is still loading in the detail view.

Needs review: @kurzee
